src/Tizen.NUI.Components/Controls/Slider.cs

```diff
@@ -1460,7 +1460,19 @@ private void UpdateState(bool isFocusedNew, bool isPressedNew)
             if (!isFocused && !isPressed)
             {
                 ControlState = ControlState.Normal;
-                thumbImage.ResourceUrl = thumbImageUrlSelector?.Normal;
+
+                if (thumbImageUrlSelector != null)
+                {
+                    thumbImage.ResourceUrl = thumbImageUrlSelector?.Normal;
```

Comment:
Once the `thumbImage.ResourceUrlSelector` is set to `thumbImageUrlSelector` then you don't have to set image resource url manually whenever the control state is changed. There is a property to use for this case: `View.EnableControlStatePropagation`.

Please refer Button's case,
https://github.com/Samsung/TizenFX/blob/master/src/Tizen.NUI.Components/Controls/Button.Internal.cs#L273

Reply:
Removed thumbImageUrlSelector variable and added 'EnableControlState = true' to Thumb ImageView.